tests/unit/test_create_dataset.py

```diff
@@ -1,33 +1,234 @@
+from functools import partial
 import json
+from unittest import TestCase
 
+from pandas import DataFrame
+from requests import HTTPError
 import responses
 
 from tamr_unify_client import Client
 from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.dataset.collection import CreationError
 
-auth = UsernamePasswordAuth("username", "password")
-unify = Client(auth)
 
+class DatasetCreationTest(TestCase):
+    def setUp(self):
```

Comment:
Could you use a [`pytest` fixture](https://docs.pytest.org/en/latest/fixture.html) here instead of `unittest.TestCase`? We already use `pytest` as our testing framework, so I'd like to use that throughout the codebase (I know there are places where we use `unittest` too, but we should strive to unify things under 1 testing framework).

Reply:
I think you and Danger should talk about this because you seem to disagree